steward/src/commands/deploy/erc20.rs

```diff
@@ -13,12 +13,17 @@ use tokio::time::sleep as delay_for;
 
 /// Deploy Erc20
 #[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Deploy ERC20 tokens to the Sommelier Chain via the Gravity Bridge.\n This command takes a Vector of Strings called denom as args.\n Options are ethereum-key and gas-multiplier which is a 64-bit floating point type, f64."
```

Comment:
Same deal here and elsewhere, we don't need to list flags/types which will be printed out automatically based on the struct.